src/component/handlers/edit/editOnCut.js

```diff
@@ -45,9 +44,9 @@ function editOnCut(editor: DraftEditor, e: SyntheticClipboardEvent<>): void {
 
   // Track the current scroll position so that it can be forced back in place
   // after the editor regains control of the DOM.
-  if (element instanceof Node) {
-    scrollPosition = getScrollPosition(Style.getScrollParent(element));
-  }
+  const editorNode = ReactDOM.findDOMNode(editor.editor);
```

Comment:
Was wondering if the previous line `Style.getScrollParent(element)` was meant to always get the 'editor' dom reference ? Is this new line having equivalent behaviour to the deleted code ? 

cc @flarnie

Reply:
@mitermayer I think the answer is 'No' to both of your questions. The line `Style.getScrollParent(element)` was meant to get the _scroll parent_ of the editor dom reference, which in many cases will be the `document.body`.

The new line is _not_ equivalent to the deleted code. In many cases it is, because the scroll parent of the cut event target will also be the scroll parent of the editor dom node. But in some cases, as described in #1427, the scroll parent is something else, leading to erroneous behaviour. The point of my pull request is to fix this and make the behaviour equivalent to https://github.com/facebook/draft-js/blob/f50c4ca9a5881763eb3028ecacf8fd56f430bc3f/src/component/base/DraftEditor.react.js#L371.